src/index.html

```diff
@@ -15,8 +15,77 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header>
+      <ul class="logo-list">
+        <li class="logo-item">
+          <a class="logo-link" href="#">
+            <img alt="Company logo" src="./images/logo.png"/>
```

Comment:
The `alt` attribute of the image doesn't describe the content of the image. It should give a more detailed description of the image. For example, if it's a logo of a specific company, mention the company's name.